WooCommerce/Classes/Tools/Currency/CurrencyFormatter.swift

```diff
@@ -190,15 +199,18 @@ public class CurrencyFormatter {
         return formatCurrency(using: humanReadableAmount,
                               at: position,
                               with: symbol,
-                              isNegative: isNegative)
+                              isNegative: isNegative,
+                              locale: locale)
     }
 
     /// Applies currency option settings to the amount for the given currency.
     /// - Parameters:
-    ///     - amount: a NSDecimalNumber representation of the amount, from the API, with no formatting applied. e.g. "19.87"
+    ///     - decimalAmount: a NSDecimalNumber representation of the amount, from the API, with no formatting applied. e.g. "19.87"
     ///     - currency: a 3-letter country code for currencies that are supported in the API. e.g. "USD"
+    ///     - locale: the locale that is used to format the currency amount string.
     ///
-    func formatAmount(_ decimalAmount: NSDecimalNumber, with currency: String = CurrencySettings.shared.currencyCode.rawValue) -> String? {
+    func formatAmount(_ decimalAmount: NSDecimalNumber, with currency: String? = nil, locale: Locale = .current) -> String? {
```

Comment:
Also here, since it's clear that `decimalAmount` is of type `NSDecimalNumber`, can we rename it `amount`?